pytest_mpl/plugin.py

```diff
@@ -686,7 +695,7 @@ def compare_image_to_hash_library(self, item, fig, result_dir, summary=None):
         hash_library_filename = compare.kwargs.get("hash_library", None) or self.hash_library
         if self._hash_library_from_cli:  # for backwards compatibility
             hash_library_filename = self.hash_library
-        hash_library_filename = (Path(item.fspath).parent / hash_library_filename).absolute()
+        hash_library_filename = _get_item_dir(item) / hash_library_filename
```

Comment:
The path should already be absolute so removing the `.absolute()` should be fine 👍